testutil/MongoSuite.go

```diff
@@ -32,37 +32,37 @@ type MongoSuite struct {
 // will return the same DBServer until TearDownDBServer is called.  In most cases, calls to DBServer() are completely
 // unnecessary, as calls to DB() will automatically instantiate the DBServer if needed.  If a test suite calls
 // DBServer(), then TearDownDBServer() MUST be called in the TearDownSuite function (or TearDownTest, if appropriate).
-func (suite *MongoSuite) DBServer() *dbtest.DBServer {
-	suite.dbServerMutex.Lock()
-	defer suite.dbServerMutex.Unlock()
+func (m *MongoSuite) DBServer() *dbtest.DBServer {
```

Comment:
It looks like the only changes in this file are renaming `suite` to `m`?  That's fine, but just FYI -- I originally called it `suite` because that's the practice that Testify takes in all its documentation/examples:
https://github.com/stretchr/testify#suite-package